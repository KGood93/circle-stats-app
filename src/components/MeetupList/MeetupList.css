.meetupTitle a {
    font-family: 'Roboto', sans-serif;
    font-size: 30px;
    color: rgb(12, 70, 94);
}

.meetupTitle a:hover {
    color: rgb(246, 125, 72);
}

.meetupTitle a:visited {
    color: rgb(12, 70, 94);
}

.meetupDate {
    font-family: 'Roboto', sans-serif;
    color: rgb(12, 70, 94);
}