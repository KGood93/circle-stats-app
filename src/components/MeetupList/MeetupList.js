import React, {Component} from 'react'
import {Link} from 'react-router-dom'
import Context from '../../context'

//Add note category on meetup section

class MeetupList extends Component {
    static contextType = Context

    render() {
        const {meetups = []} = this.context
        const meetup = meetups
        //console.log(meetups)
        return (
            meetup.map((meetupDetails, index) => {
                return (
                    <div className="meetupDiv" key={index}>
                        <h2 className="meetupTitle">
                            <Link to={`/launch/${meetupDetails.meetup_id}`}>{meetupDetails.meetup_name}</Link>
                        </h2>
                        <h4 className="meetupDate">{meetupDetails.meetup_note}</h4>
                </div>
                )
            })
        )
    }
}

export default MeetupList