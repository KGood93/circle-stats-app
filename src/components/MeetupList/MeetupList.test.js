import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import MeetupList from './MeetupList';

describe('MeetupList Component', () => {
    it(`renders without crashing`, () => {
        const wrapper = shallow(<MeetupList />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});