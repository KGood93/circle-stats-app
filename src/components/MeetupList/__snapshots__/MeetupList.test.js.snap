// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MeetupList Component renders without crashing 1`] = `null`;
